shared/pingpong/pingpong.go

```diff
@@ -403,26 +411,40 @@ func waitPendingTransactions(accounts map[string]*pingPongAccount, client libgoa
 	return nil
 }
 
-func (pps *WorkerState) refreshAccounts(accounts map[string]*pingPongAccount, client libgoal.Client, cfg PpConfig) error {
+func (pps *WorkerState) refreshAccounts(client libgoal.Client, cfg PpConfig) error {
+	pps.accountsMu.Lock()
+	addrs := make([]string, len(pps.accounts))
```

Comment:
```suggestion
	addrs := make([]string, 0, len(pps.accounts))
```
and `append()` below